app/helpers/application_helper.rb

```diff
@@ -364,4 +364,17 @@ def has_out_of_date_configuration?(service)
     return unless service
     service.pending_affecting_changes?
   end
+
+  def rails_asset_url_tag
+    javascript_tag("window.rails_asset_host = '#{rails_asset_host_url}'", { type: "text/javascript"})
+  end
+
+  def rails_asset_host_url
+    asset_host_enabled = Rails.configuration.asset_host.present?
+    asset_host_url = Rails.configuration.three_scale.asset_host.presence
+
+    return '' unless asset_host_enabled && asset_host_url
+
+    "#{request.protocol}#{asset_host_url}"
```

Comment:
It's not checking the string twice because only `asset_host_url` is a string, `asset_host_enabled` is a boolean. [This commend](https://github.com/3scale/porta/pull/3072#discussion_r1007756486) explains why both are required.